packages/terra-status-view/src/terra-dev-site/doc/example/TextWrappingExample.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Button from 'terra-button';
+import IconDiamond from 'terra-icon/lib/icon/IconDiamond';
+import StatusViewTemplate from '../example/StatusViewTemplate';
+
+const TextWrappingExample = (
+  <StatusViewTemplate
```

Comment:
Should we update these to actually StatusView instead of a template example component? Right now showing the example source code that uses a template provide much insights to consumers regarding how to use it

Reply:
Updated: https://github.com/cerner/terra-core/pull/2130/commits/ac8e9c25a6474a23ee6a9e73ac8e43e80743f4fc